pushy/src/main/java/com/turo/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -671,6 +717,14 @@ public String buildWithMaximumLength(final int maximumPayloadSize) {
                 aps.put(THREAD_ID_KEY, this.threadId);
             }
 
+            if (this.summaryArgument != null) {
+                aps.put(SUMMARY_ARGUMENT, this.summaryArgument);
```

Comment:
Do we want these in the `aps` dictionary, or in the `aps.alert` dictionary?

Reply:
In `aps.alert`, moved them in ed6aed6.